crates/server/src/server/server_future.rs

```diff
@@ -72,6 +77,17 @@ impl<T: RequestHandler> ServerFuture<T> {
 
                     let src_addr = message.addr();
                     debug!("received udp request from: {}", src_addr);
+
+                    // verify that the src address is safe for responses
+                    if let Err(e) = sanitize_src_address(src_addr) {
+                        warn!(
+                            "address can not be responded to {src_addr}: {e}",
+                            src_addr = src_addr,
```

Comment:
Since rust 1.58.0 this is no longer needed, it is now more a question of what minimum version to support:
https://blog.rust-lang.org/2022/01/13/Rust-1.58.0.html#captured-identifiers-in-format-strings

Reply:
According to our 3 release look back policy, 1.57 is currently the msrv we could upgrade to. In six weeks we could come back and update this.